app/features/bookmarks/bookmark.service.server.ts

```diff
@@ -0,0 +1,35 @@
+import { getConfigEntry } from "~/common/config.server";
+import { request } from "~/common/request.server";
+import type { Bookmark } from "./bookmark.types";
+
+const apiRequest = request(getConfigEntry("API_URL"));
```

Comment:
Totally separate topic, I thought of a hole in this pattern of initializing as a singleton....

All of our API calls are going to need an access token included. In Remix, that access token is probably going to be pulled off a cookie on the `Request`.  There for we'd probably want to create an instance of this `apiRequest` at the start of the request life cycle so that we can pass in the access token.  

If we don't, we'd probably have to pass the access token to every single service method.